src/methods.rs

```diff
@@ -135,17 +135,20 @@ mod shared_impls {
 }
 
 #[cfg(feature = "any")]
-pub use any::new as any;
+pub use any::request as any;
 
 #[cfg(feature = "any")]
 mod any {
     use super::*;
     use std::marker::PhantomData;
 
-    pub fn new<T, E>(method_name: &str, params: serde_json::Value) -> RpcAnyRequest<T, E>
+    pub fn request<T: AnyRequestResult>(
```

Comment:
Do we actually need custom deserialization on response / error for `any` methods? I suppose it's just the request that should allow for that flexibility. If not, then instead of an intermediate `AnyRequestResult` here, I'll just use an `RpcMethod`..

Reply:
I think it is fine to keep custom deserialization and if someone wants to skip strict type deserialization, they should be able to just use `serde_json::Value` type here.